### DISCUSSION TITLE
台灣缺電所以不適合發展電動車？

### DISCUSSION THREAD

**Discussion by Patsy17** [1101-GS4719-C/Python-Community-GS4719-C#1822] [General]
幾乎全球都在發展電動車，減少更多的碳排放與節能，
台灣目前電力供應情況並不算充足，在這種狀況下，持續發展電動車對台灣來說是可行的嗎？
我查過ㄧ些文章指出缺電問題，只要電動車車主利用離峰時段充電（晚間時段），並非無法克服，並不會造成太大的電網壓力。
但有其他聲音表示台灣電動車最大問題其實並非缺電，而是並非所有車主家中皆有車位，可能都是路邊停車或租在停車場內，沒有充足的電樁車位。

資料來源：https://tw.news.yahoo.com/台灣缺電所以不適合發展電動車-網友分析真正原因其實是這個-070852450.html

**Comment by B1nn9** [comment]
我覺得很可惜的一件事是電動車原是為了減少碳排，但臺灣電力來源現階段主要是火力發電，而利用如此發電方式所得的電來供應電動車使用，顯然會與原先的環保理念背道而馳😪
